mkdocs.yml

```diff
@@ -170,7 +170,10 @@ plugins:
         # rm: Romansh
         # ja: Japanese
         # gl: Galician
-        pt: Portuguese
+        pt: 
+          name: Portuguese
+          site_name: Documentação do Ecossitema QField
+          # site_description: plugin i18N don't accept site_description - Warning: plugins.i18n.languages.pt' unsupported options: site_description
```

Comment:
@m-kuhn probably we need to do a PR here to accept the site_description:
https://github.com/ultrabug/mkdocs-static-i18n/blob/67a324bd019d9a0f3b31afb652e06b8e0af64095/mkdocs_static_i18n/structure.py#L50

If they accept we can bump the version of the mkdocs-static-i18n extension in the requirements of qfield docs

Reply:
@jonnyforestGIS good idea, do you want to go ahead?

Reply:
I will try. I Already fork the extension And I'm changing the code.

Reply:
Hello, there is probably no need to fork the extension as the site_description key might be added with the upcoming https://github.com/ultrabug/mkdocs-static-i18n/pull/216. I'd suggest not using the key until they merge that PR. I'll try to make sure that it adds the desired key before merging as it's a common use-case.

Reply:
Thank @why-not-try-calmer for the feedback. I will stop the changes and we wait to this refactoring from mkdocs-static-i18n and then we can add this PR with site_description included.

Reply:
@why-not-try-calmer what is the status of this PR? Should I close it?

Reply:
> @why-not-try-calmer what is the status of this PR? Should I close it?

Short answer is No, because this PR is a reminder that there's a crucial feature that needs to be added for benefit of about every language except English.

Explanation: Apparently the situation is a little bit difference as https://github.com/ultrabug/mkdocs-static-i18n/pull/216 to be merged has evolved into https://github.com/ultrabug/mkdocs-static-i18n/pull/220 and from what I see more back-and-forth will be needed. So there are three options depending on @jonnyforestGIS's patience and interests:
1. get in touch with the people upstream working on the aforementioned PR, see if a smaller patch for site_description can be extracted and merged soon
2. do the patch yourself @jonnyforestGIS, but then if upstream merges soon your work might be a little be wasteful.

Reply:
Quick update status:

Accordingly, with the developer, he answers:

_yes I have many things going on at the same time so I'm pushing this PR in the backlog._

_Another reason is that we are working on the MkDocs 1.5 release which will change some internals so I'd rather make this PR target the latest MkDocs version instead._

_I should spend again some time on this PR by the end of the month!_

So if I understand well probably the mkdocs will need to upgrade to 1.5 as well the new version of the plugin in the qfield-docs.

Reply:
Yes, that's how I understand it too.

Reply:
@why-not-try-calmer  can we close this? after updating PR #356 probably doesn't make sense to approve this PR.